synapse/handlers/oidc.py

```diff
@@ -413,6 +509,14 @@ def _uses_userinfo(self) -> bool:
             or self._user_profile_method == "userinfo_endpoint"
         )
 
+    @property
+    def issuer(self) -> str:
```

Comment:
Docstrings for both these properties please!

Reply:
Done